### DISCUSSION TITLE
Where can I find open_agb_firm.firm file?

### DISCUSSION THREAD

**Discussion by N00bes** [profi200/open_agb_firm#188] [Q&A]
I know... I'm an idiot. But I just can't find it! I go to the latest release page, and the files I see are in the assets section, but the only ones there is the .zip folder (I can't find .firm within),  .7z file, and tar.gz. Where is the file that I need?

**Comment by profi200** [comment]
Right in the root when opening the .7z archive. And no, only unpacking the FIRM file is not enough. Unpack the 3ds folder too.
